Setup/Migrations/ReplaceFitInMailbox.php

```diff
@@ -89,7 +87,7 @@ public function updateCatalogProductEntity(): void
      */
     private function calculatePercentToValue($entity): float
     {
-        return round((100 / $entity['value']));
+        return (null !== $entity['value']) ? round((100 / $entity['value'])) : $entity['value'] = 0;
```

Comment:
Volgens de styleguide mogen we geen toewijzingen doen in statements die returnen of conditioneel zijn. Ik betwijfel ook of dat is wat je bedoelde hier...